stacks-core/release/check-release/action.yml

```diff
@@ -0,0 +1,80 @@
+## Composite action to check if the branch is a release tag or not
+name: Check Release
+
+inputs:
+  tag:
+    description: "The branch name against which the step runs"
+    required: false
+    default: ""
+
+outputs:
+  node_tag:
+    description: "The node release tag, if there is one (empty otherwise)."
+    value: ${{ steps.check_release.outputs.node_tag }}
+  node_docker_tag:
+    description: "The node release tag for docker, if there is one (empty otherwise)."
+    value: ${{ steps.check_release.outputs.node_docker_tag }}
+  signer_tag:
+    description: "The signer release tag, if there is one (empty otherwise)."
+    value: ${{ steps.check_release.outputs.signer_tag }}
+  signer_docker_tag:
+    description: "The signer release tag for docker, if there is one (empty otherwise)."
+    value: ${{ steps.check_release.outputs.signer_docker_tag }}
+  is_node_release:
+    description: "True if the branch is a release one, false otherwise."
+    value: ${{ steps.check_release.outputs.is_node_release }}
+  is_signer_release:
+    description: "True if the branch is a release one, false otherwise."
+    value: ${{ steps.check_release.outputs.is_signer_release }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set Release Outputs
+      id: check_release
+      shell: bash
+      run: |
+        branch_name=${{ inputs.tag }}
+
+        stacks_core_version_regex="([0-9].[0-9].[0-9].[0-9].[0-9]+(-rc[0-9]+)?)"    # matches x.x.x.x.x || x.x.x.x.x-rcx
+        signer_version_regex="([0-9].[0-9].[0-9].[0-9].[0-9].[0-9]+(-rc[0-9]+)?)"   # matches x.x.x.x.x.x || x.x.x.x.x.x-rcx
+
+        stacks_core_regex_base="release/"
+        signer_regex_base="release/signer-"
+
+        stacks_core_regex="${stacks_core_regex_base}${stacks_core_version_regex}"
+        signer_regex="${signer_regex_base}${signer_version_regex}"
+
+        node_tag=""
+        node_docker_tag=""
+        signer_tag=""
+        signer_docker_tag=""
+        is_node_release=false
+        is_signer_release=false
+
+        if [[ "$branch_name" =~ ^${stacks_core_regex}$ || "$branch_name" =~ ^${signer_regex}$ ]]; then
```

Comment:
can you an else clause here for verbosity? as-is it should work fine, but i'd prefer to see an else here that halts the workflow in case it got this far with an unrecognized branch name. 

potenitally it could be put into the `case` entirely, and the conditional removed. 

ex:

```
case ${branch_name} in
  release/signer-[0-9]*)
    ...
  release/[0-9]*)
    ...
    *)
       exit 1
 ```